### PR TITLE
Replace MK80 in squad vendors with MK80 pistol case

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
## About the PR
add MK80 pistol case

also adds smart pistol case to smart pistol vendor for cm13 parity

## Why / Balance
cm13 parity and in todo list

## Media
image

image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl:
- add: Added MK80 hand cannon pistol case

### CODE REVIEW

File: Resources/Prototypes/_RMC14/Entities/Objects/Weapons/specialist_loadouts.yml [hunk 1]
```diff
@@ -94,9 +93,7 @@
     - id: RMCHandsM3G4
     - id: RMCArmorM3G4
     - id: RMCArmorHelmetM3G4 # hugger protection
-    - id: CMWeaponPistolMK80
-    - id: CMMagazinePistolMK80
-    - id: CMMagazinePistolMK80
+    - id: RMCMK80Case
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Objects/Weapons/specialist_loadouts.yml:96]: This one is actually supposed to be the three items individually
```suggestion
    - id: CMWeaponPistolMK80
    - id: CMMagazinePistolMK80
    - id: CMMagazinePistolMK80
```

File: Resources/Prototypes/_RMC14/Entities/Objects/Weapons/specialist_loadouts.yml [hunk 2]
```diff
@@ -41,8 +41,7 @@
     - id: CMMagazineSniperM96S # TODO rmc14 CMMagazineSniperM96SIncendiary
     - id: CMMagazineSniperM96S # TODO rmc14 CMMagazineSniperM96SFlak
     - id: CMBackpackSniper
-    - id: CMWeaponPistolMK80
-    - id: CMMagazinePistolMK80
+    - id: RMCMK80Case
```
Comment (DrSmugleaf) [inline Resources/Prototypes/_RMC14/Entities/Objects/Weapons/specialist_loadouts.yml:44]: Same with this
```suggestion
    - id: CMWeaponPistolMK80
    - id: CMMagazinePistolMK80
    - id: CMMagazinePistolMK80
```

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by Dutch-VanDerLinde** [comment]
requested changes done

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**